package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lodev09/react-native-true-sheet",
-  "version": "1.0.3",
+  "version": "1.0.4",
```

Comment:
Can you revert this please? We'll let the release automatically update this.

Reply:
done